webcompat/views.py

```diff
@@ -155,8 +155,22 @@ def show_issues():
     return render_template('list-issue.html', categories=categories)
 
 
-@app.route('/issues/new', methods=['POST'])
+@app.route('/issues/new', methods=['GET', 'POST'])
 def create_issue():
+    """Creates a new issue
+
+    GET will return an HTML response for reporting issues
+    POST will create a new issue
+    """
+    if request.method == 'GET':
+        bug_form = IssueForm()
```

Comment:
This starts to feel a bit redundant w/ the index view, but I'm not sure how to make it more DRY. 🍤